compiler/src/iree/compiler/Dialect/LinalgExt/Utils/Utils.cpp

```diff
@@ -750,6 +750,9 @@ bool isaHorizontallyFusedContraction(linalg::LinalgOp linalgOp) {
   };
   llvm::SmallDenseSet<unsigned> lhsDims = getResultDims(lhsIndexingMap);
 
+  // Check that all the horizontally fused gemms have common k-dims. M and N
+  // dims are already known consistent since they are what the LHS has.
```

Comment:
The LHS only has M, do we need a check for N still?

Reply:
typo. Fixed.